nikolay.braslavets/hw01/1.rb

```diff
@@ -0,0 +1,7 @@
+array = Array.new(3) { Array.new(3) { rand(-10..10) } }
+puts  "We created new array: #{array}"
+
+array.flatten.select(&:positive?).each { |number| puts "Find positive number is: #{number}" }
+
+sumPositive = array.flatten.select(&:positive?).sum
```

Comment:
in ruby we call local variables using snake_case
https://github.com/rubocop-hq/ruby-style-guide#snake-case-symbols-methods-vars
Also you can check full ruby style guide here
https://github.com/rubocop-hq/ruby-style-guide

Reply:
```suggestion
sum_positive = array.flatten.select(&:positive?).sum
```